debian/python3-bocs.bocs.udev

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="tty", ACTION=="add", KERNELS=="1-1.2", SYMLINK+="BOCS_SENSOR_ARRAY"
```

Comment:
I think there are better ways to do this - I rushed this bit because I needed to get it to work on short notice. This requires that the sensor array logger is connected to a specific USB socket, but I believe there are other ways to achieve this that would allow it to be connected to any USB socket.

Reply:
Ok I'll have a look at this next time I'm in the lab, which will be next week